### ISSUE TITLE
Use latest Connexion

### ISSUE THREAD

**Issue by hjacobs** [zalando-zmon/zmon-slo-metrics#73]
Can we upgrade to the latest Connexion release (https://github.com/zalando/connexion/releases/tag/1.1.14) or are there any impediments?

**Comment by mohabusama** [comment]
Yes, we should upgrade and test if anything is broken. Just didn't want to change many variables during refactoring
